src/main/scala/io/citrine/theta/Stopwatch.scala

```diff
@@ -58,7 +62,197 @@ object Stopwatch {
+    * @param falseNegative         False negative rate. (Also known as the probability of observing a Type II error.)
+    *                              thrown because the null hypothesis could not be tested at the specified
+    *                              false positive and false negative rates. In which case, either the maximum number
+    *                              of runs should be increased or the FP/FN rates should be relaxed.
+    * @param minimumTimeDifference Minimum time difference (in seconds) required to conclude the runtime is slower
+    *                              than the specified time.
+    * @param acceptableEffectSize  If defined, the test will exit if the effect size exceeds this value.
+    *                              Large, medium and small effects are given by 0.8, 0.5 and 0.2, respectively.
+    *                              Units are defined in terms of standard deviations of the mean. The effect size
+    *                              is directional. If positive, the test will exit early if the mean execution time
+    *                              is clearly slower than the given `time`. If negative, the test will exit early if
+    *                              the execution time is clearly faster than the specified time. Note, however,
+    *                              that ''clearly'' in this context is defined by the effect size you specify.
+    * @tparam R Return type of the execution block
+    * @return Whether the function block is faster than the specified time.
+    */
+  def isSlowerThan[R](
+                       block: => R, time: Double,
+                       minRun: Int = 4, maxRun: Int = 64,
+                       falsePositive: Double = 0.05, falseNegative: Double = 0.20,
+                       minimumTimeDifference: Double = 0.0,
+                       acceptableEffectSize: Option[Double] = None
+                     ): Boolean = {
+    validateInputs(minRun, maxRun, falsePositive, falseNegative)
+
+    val times = mutable.ListBuffer.empty[Double]
+    var iteration: Int = 0
+    while (iteration < minRun || iteration < maxRun) {
+      val start = System.nanoTime()
+      block
+      val thisTime: Double = (System.nanoTime() - start) * 1.0e-9 // convert to seconds
+      times.append(thisTime)
+      iteration += 1
+      val sampleSize = times.size
+      if (sampleSize > 1 && iteration >= minRun) {
+        val sampleMean = calcHodgesLehmannMedian(times)
+        val sampleMAD = calcMAD(times)
+        val sampleStd = convertMADtoStd(sampleMAD)
+
+        val tDist = new TDistribution(sampleSize - 1)
+        val tAlpha = tDist.inverseCumulativeProbability(falsePositive)
+        val tBeta = tDist.inverseCumulativeProbability(falseNegative)
+        val delta = sampleMean - time - minimumTimeDifference
+        val tRequiredSampleSize = Math.pow(sampleStd * (tAlpha + tBeta) / delta, 2)
```

Comment:
This symmetric treatment of `tAlpha` and `tBeta` wrt `signum(sampleMean - time)` strikes me as maybe not right.  Do we get the expected behavior when FP is very low, FN is very high, and sampleMean << time or sampleMean >> time ?